internal/server/v1/observations/golden/bulk_series_linked/preferred_FRA_AA2_2016.json

```diff
@@ -93,136 +93,6 @@
             }
           ]
         },
-        {
-          "entity": "nuts/FR10",
```

Comment:
similar here, seems suspicious diff

Reply:
cl/638945277 - Revert NUTS<N> to AA<N> mapping.